packages/firestore/src/local/indexeddb_mutation_queue.ts

```diff
@@ -190,7 +190,7 @@ export class IndexedDbMutationQueue implements MutationQueue {
     );
 
     this.metadata.lastAcknowledgedBatchId = batchId;
-    this.metadata.lastStreamToken = validateStreamToken(streamToken);
+    this.metadata.lastStreamToken = convertStreamToken(streamToken);
```

Comment:
I was kinda surprised that this works as is, but didn't dig to much deeper since my main goal was just to get the tests passing. I filed a bug to do this cleanup later (https://b.corp.google.com/issues/78771403).